include/gepetto/viewer/corba/windows-manager.hh

```diff
@@ -215,6 +215,7 @@ namespace graphics {
             virtual bool setWireFrameMode(const char* nodeNameCorba, const char* wireFrameModeCorba);
             virtual bool setLightingMode(const char* nodeNameCorba, const char* lightingModeCorba);
             virtual bool setHighlight(const char* nodeNameCorba, int state);
+            virtual bool setAlpha(const char *nodeNameCorba, const value_type* integerValue);
```

Comment:
Prefer passing a reference, as you only pass one value. Pointers to pass constant single (non-array) base type (int, float...) are not as safe as passing reference.